tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -35,6 +35,11 @@
         )
     ]
 )
+@pytest.mark.skipif(
```

Comment:
since we are using it in few places, i think this can move to custom marks.

Reply:
@vasukulkarni Turns out we already have one actually, `skipif_aws_i3`. I'll use this instead!

Reply:
Done!